src/docs/community/rebrand.mdx

```diff
@@ -0,0 +1,27 @@
+import { Divider, FAQExpander } from '../../app/components';
+
+<!-- KEYWORDS: faq question answers -->
+
+# PX Blue is Rebranding
+
+In Q4 of 2021, we will be rebranding our design system from PX Blue to Brightlayer UI. This rebranding effort will impact all resources that PX Blue currently provides, including:
+
+-   NPM Packages
+-   Github Repositories
+-   Figma Stickersheet
+-   Documentation
+-   This documentation site & URL
+
+## Steps Required to Update
+
+### For Developers
+
+All [`@pxblue` NPM packages](https://www.npmjs.com/~px-blue) packages will be renamed from `@pxblue/*` to `@brightlayer-ui/*`.
+
+Make sure to update to the latest version of each `@pxblue` package before the rebranding occurs to minimize the amount of work required when updating to `@brightlayer-ui`.
+
+Only the latest **2 major versions** for each `@pxblue` package will be migrated over to using the `@brightlayer-ui` namespace.
+
+The steps required to update will vary between packages.
```

Comment:
"Vary between packages" — so where should the users be looking at? Will there be changelog items? Is it on NPM, somewhere in the readme? Or just console logs? You will want to set up users' expectation here.